doc/source/reference/c-api.coremath.rst

```diff
@@ -81,7 +81,22 @@ Useful math constants
 ~~~~~~~~~~~~~~~~~~~~~
 
 The following math constants are available in npy_math.h. Single and extended
-precision are also available by adding the F and L suffixes respectively.
+precision are also available by adding the f and l suffixes respectively, with the exceptions of the following: -
```

Comment:
This doesn't look right, you need to use rst markup.

Reply:
I would not list all the versions either, just the bare types with with a note that they use the capitalized versions.